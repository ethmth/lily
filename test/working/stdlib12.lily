
let a::int = range(4, 9)

let b::int = range(17, 24)

a.remove(6)

a.reverse()

#a.print()

print(concatenate(a, b))