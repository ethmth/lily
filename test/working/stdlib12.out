[8, 7, 5, 4, 17, 18, 19, 20, 21, 22, 23]
